### PR TITLE
Change Some Names

### PR DESCRIPTION
**Pull request by rboston628**
## Description of work

Several of the views and workflows had names which (in my humble opinion) were sort of confusing.  This is a minimal update to change the names of the files.

Also, there was a slight error inside the normalization workflow not caught in PR #250 , which is fixed here.  At the tweaking stage, the grouping list was being populated with the default groupings.

## Explanation of work

All view names within the DiffCal workflow now beign with `DiffCal`, then have some description that follows:
- `DiffCalRequestView`
- `DiffCalAssessmentView`
- `DiffCalSaveView`

All view names within the Normalization workflow now begin with `Normalization`, then have some description that follows:
- `NormalizationRequestView`
- `NormalizationTweakPeaksView`
- `NormalizationSaveView`

I changed these names everywhere I could find them in the code.

`BackendRequestView` now has a default value in the `_labeledField` method, which will create a `QLineEdit` with a label.

### Dev testing

Run through the diffcal process using 46680, make sure you can save.

Run through the normalization process using 46680.  Double-check the available groupings on the tweaking stage.  Make sure you can save.

### CIS testing

This is an internal issue and does not need CIS testing.

## Link to EWM item
None.

### CODE REVIEW

File: src/snapred/resources/dev.yml
Comment (rboston628) [inline src/snapred/resources/dev.yml:1]: This should be ignored by the gitignore.  Why is it here???

**Comment by walshmm** [comment]
I'd probably reccomend waiting on other PR's before merging this one

**Comment by rboston628** [comment]
My hope was this would make PR #251 easier to merge, if I merged this first

**Review by darshdinger — APPROVED** [review]
Looks good!

**Pull request merged**